config-templates/ibm/ibm-csi-block/1.4.0/storage-class-parameters.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+	"description": "The name of the storage class that is created.",
+	"displayname": "Storage Class name",
+	"name": "sc-name",
```

Comment:
Change "sc-name" to  "name" . we have some checks over it

Reply:
Also add new variable for `is-default-class`

eg 
```{
	  "description": "Specify 'true' or 'false' to make the created storage class the default class.",
	  "displayname": "Default",
	  "name": "is-default-class",
	  "default": "false",
	  "required": "false",
	  "category": "config",
	  "type": "text"
	},
	```

Reply:
done